homeassistant/components/recorder/migration.py

```diff
@@ -1089,6 +1089,11 @@ def _apply_update(  # noqa: C901
             "states",
             [f"last_reported_ts {_column_types.timestamp_type}"],
         )
+    elif new_version == 44:
```

Comment:
This will only work if we never alter the schema again. It needs to change the check in the other place instead to only do this after schema migration is done